website/docs/concurrency_control.md

```diff
@@ -2,105 +2,126 @@
 
 ```
 hoodie.write.concurrency.mode=optimistic_concurrency_control
-hoodie.cleaner.policy.failed.writes=LAZY
 hoodie.write.lock.provider=<lock-provider-classname>
 ```
 
-There are 4 different lock providers that require different configurations to be set.
+| Config Name| Default| Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| --------------------------------------------------------------------------------- | ------------------------ |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| hoodie.write.concurrency.mode | SINGLE_WRITER (Optional) | <u>[Concurrency modes](https://github.com/apache/hudi/blob/c387f2a6dd3dc9db2cd22ec550a289d3a122e487/hudi-common/src/main/java/org/apache/hudi/common/model/WriteConcurrencyMode.java)</u> for write operations.<br />Possible values:<br /><ul><li>`SINGLE_WRITER`: Only one active writer to the table. Maximizes throughput.</li><li>`OPTIMISTIC_CONCURRENCY_CONTROL`: Multiple writers can operate on the table with lazy conflict resolution using locks. This means that only one writer succeeds if multiple writers write to the same file group.</li></ul><br />`Config Param: WRITE_CONCURRENCY_MODE` |
+| hoodie.write.lock.provider    | org.apache.hudi.client.transaction.lock.ZookeeperBasedLockProvider (Optional)                      | Lock provider class name, user can provide their own implementation of LockProvider which should be subclass of org.apache.hudi.common.lock.LockProvider<br /><br />`Config Param: LOCK_PROVIDER_CLASS_NAME`<br />`Since Version: 0.8.0`                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
to enable multi-writers, users should also set hoodie.cleaner.policy.failed.writes to LAZY 

I feel, we should simplify the usage and enable the lazy internally when OCC is enabled. but guess as per master, users have to explicitly set the clean failed writes policy

Reply:
oh I see, you have listed above. but not in the table. should we include this config as well

Reply:
> to enable multi-writers, users should also set hoodie.cleaner.policy.failed.writes to LAZY
> 
> I feel, we should simplify the usage and enable the lazy internally when OCC is enabled. but guess as per master, users have to explicitly set the clean failed writes policy

Yes +100.

Reply:
> Will add it back.